packages/remediations/src/steps/reviewSystems.js

```diff
@@ -52,22 +60,17 @@ const ReviewSystems = ({ issues, systems, registry, ...props }) => {
         }
     };
 
-    const onSelect = (selected) => {
-        input.onChange(selected);
-    };
-
     const onSelectRows = (value) => {
         dispatch({
             type: TOGGLE_BULK_SELECT,
             payload: value
         });
     };
 
-    const { selected, loaded, rows } = useSelector(({ entities }) => ({
-        selected: entities?.selected || [],
-        loaded: entities?.loaded,
-        rows: entities?.rows || []
-    }));
+    if (!isEqual(selectedRef.current, selected)) {
```

Comment:
maybe you don't have to have ref, just use the comparison array in useEffect hook. 🤔